run_bot.py

```diff
@@ -9,6 +9,22 @@
 # the channel id to restrict sending messages to
 botspam_channel_id = int(sys.argv[1])
 
+def generate_message(serialized_user):
```

Comment:
I think it would be better if this just took a user ID and handled all of the markov chain loading and generating logic, include file path.